InnerEye/Common/common_util.py

```diff
@@ -421,3 +428,75 @@ def change_working_directory(path_or_str: PathOrString) -> Generator:
     os.chdir(new_path)
     yield
     os.chdir(old_path)
+
+
+def merge_conda_files(
```

Comment:
I have removed conda merging from `hi-ml` because it turned into a source of trouble. We should discuss if we can do the same for InnerEye.

Reply:
Agreed, will open an issue to remove this in the future but for now I think it's worth keeping in this PR to avoid making too many changes at once